Automattic/wp-calypso
# Mailing Lists: Remove category for Jetpack agency onboarding notifications

This change removes Calypso (UI) support for the mailing list category 'Jetpack Agencies Pro Onboarding' used for Jetpack agency onboarding notifications. 

# Testing Instructions
The backend work on removing the mailing list category for Jetpack Agency Pro Onboarding will be completed via D116519.

- Visit [this unsubscribe URL](http://calypso.localhost:3000/mailing-lists/unsubscribe?category=jetpack_agency_pro_onboarding&email=codefourcomics%40gmail.com&hmac=37190afea942b00d5838728979103e9a). Confirm that the category name and the description no longer appears.
<img width="1481" alt="CleanShot 2023-07-19 at 21 15 00@2x" src="https://github.com/Automattic/wp-calypso/assets/135139690/9eae7641-aa49-4b19-8d54-9e5ef7a6c1d0">
